python/ray/autoscaler/kubernetes/operator_configs/operator_cluster_scoped.yaml

```diff
@@ -47,6 +47,9 @@ spec:
         imagePullPolicy: Always
         image: rayproject/ray:nightly
         command: ["ray-operator"]
+        env:
+        - name: AUTOSCALER_MAX_NUM_FAILURES
+          value: "999999"
```

Comment:
nit: this is not really infinity

Reply:
well...with the default `AUTOSCALER_UPDATE_INTERVAL_S` of 5, it would take about two months of consecutive autoscaler failures for this number to be exceeded.
But it does look silly -- updated it to "inf". 
`env_integer` in `autoscaler/_private/constants` updated to interpret that as `sys.maxsize` which is a bigger non-infinite integer.